### PR TITLE
Bugfix: cleanup didn't unswizzle classes that had already been swizzled

### PR DESCRIPTION
**Pull request by fpillet**
This would happen when using other code that itself swizzles classes (i.e. ReactiveCocoa). For example, if a rac_signalForSelector was called on an object then later an aspect hook was installed and further removed, we would end up in a situation where Aspect's class swizzling (in-place swizzling of -forwardInvocation:) would never be ununstalled when the last hook was removed, later wrecking havoc with RAC's own use of -forwardInvocation.

In short, this pull request fixes hair-pulling crashes.

### CODE REVIEW

**Comment by fpillet** [comment]
To make things crystal clear, this fixes properly reverts the -forwardInvocation swizzling done by: 

https://github.com/steipete/Aspects/blob/master/Aspects.m#L358

**Comment by steipete** [comment]
lgtm - thanks!
